server/hotrod/src/main/java/org/infinispan/server/hotrod/ClientListenerRegistry.java

```diff
@@ -302,7 +298,11 @@ protected long getEventId(CacheEntryEvent event) {
       protected final ClientEventType targetEventType;
       protected final Cache cache;
 
-      BlockingQueue<Events.Event> eventQueue = new LinkedBlockingQueue<>(100);
+      final int highWaterMark = 100;
+      final int lowWaterMark = 60;
```

Comment:
How did you come to these values? Benchmarking?

Reply:
Just made up numbers at this point, similar to the old queue size.